.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+python:
+    - "2.7"
+before_install:
+    - "export DISPLAY=:99"
+    - "sh -e /etc/init.d/xvfb start"
+    - "git fetch origin master:pylint-basis"
+    - "git diff pylint-basis --name-only | grep \\\\.py$ > CHANGED_PY_FILES || echo No changed .py files"
```

Comment:
You can pass glob pattern to git directly:

    git diff --exit-code --name-only -- '*.py' > CHANGED_PY_FILES

This might invert the exit codes, not sure, so you might need && instead of ||.